README.md

```diff
@@ -2,20 +2,12 @@
 
 [![Travis Build Status](https://travis-ci.com/logstash-plugins/logstash-output-example.svg)](https://travis-ci.com/logstash-plugins/logstash-output-example)
 
-This is a plugin for [Logstash](https://github.com/elastic/logstash).
-
-It is fully free and fully open source. The license is Apache 2.0, meaning you are pretty much free to use it however you want in whatever way.
+> This project is developed and maintained by Dynatrace R&D.
```

Comment:
This means there is no warranty or support?

Reply:
These details are covered globally in the organization description.

Reply:
it does not mean that

Reply:
/cc @arminru